static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -647,11 +647,15 @@ function ProfileDigest() {
             <Count value={data?.['count()'] as number} />
           )}
         </div>
-      </div>
+      </ProfileDigestColumn>
     </ProfileDigestHeader>
   );
 }
 
+const ProfileDigestColumn = styled('div')`
```

Comment:
could potentially just import `NumberContainer` from https://github.com/getsentry/sentry/blob/master/static/app/utils/discover/styles.tsx#L20

Reply:
Oh I see. I noted this into a todo for polishing, I'll get the main functionality wired up and do this in the end, thanks!